### PR TITLE
More small fixes

### PR DESCRIPTION
**Pull request by SaphiraSkies**

### CODE REVIEW

**Comment by SaphiraSkies** [comment]
Also taking a moment to celebrate 100 pull requests on our game here, I suppose lol.

**Comment by SaphiraSkies** [comment]
With the last fixes I've added on here, I'm hoping the entire top floor of the house works now.

Batteries, flashlight, polaroid should all work. All conditionals should work, including the paintings.

Let me know if you find any issues, of course, but I'm moving on to the ground floor.

paintings

File: Language_Parser/actionVerbs.py [hunk 1]
```diff
@@ -34,11 +34,18 @@ def examine(info):
         print(errorString)
         return
 
+    game = info["Game"]
+    player = game.getPlayer()
+    room = player.getLocation()
+    allItems = player.getInventory() + room.getAccessibleItems()
```
Comment (CharlesBeise) [inline Language_Parser/actionVerbs.py:40]: allItems is being created twice, once here and again in lines 51-53.

**Review by CharlesBeise — CHANGES_REQUESTED** [review]
Looks good! Only issue is the duplication of "allItems" in actionVerbs

File: Language_Parser/actionVerbs.py (hunk 1 above)
Comment (SaphiraSkies) [reply Language_Parser/actionVerbs.py:40]: Thanks, fixed it.

**Review by CharlesBeise — APPROVED** [review]

**Pull request merged**